pkg/config/config-map_test.go

```diff
@@ -66,17 +61,7 @@ var _ = Describe("ConfigMap", func() {
 	})
 
 	It("Should not create a new config map iso disk without a Disk device", func() {
-		vmi := api.NewMinimalVMI("fake-vmi")
-		vmi.Spec.Volumes = append(vmi.Spec.Volumes, v1.Volume{
-			Name: "configmap-volume",
-			VolumeSource: v1.VolumeSource{
-				ConfigMap: &v1.ConfigMapVolumeSource{
-					LocalObjectReference: k8sv1.LocalObjectReference{
-						Name: "test-config",
-					},
-				},
-			},
-		})
+		vmi := libvmi.New()
```

Comment:
You are missing `VolumeSource` here

Reply:
Since this is a "wrong" configuration, it is fine, to me, to directly manipulate the spec:
```
		vmi.Spec.Volumes = append(vmi.Spec.Volumes, v1.Volume{
			Name: "configmap-volume",
			VolumeSource: v1.VolumeSource{
				ConfigMap: &v1.ConfigMapVolumeSource{
					LocalObjectReference: k8sv1.LocalObjectReference{
						Name: "test-config",
					},
				},
			},
		})
```